lib/mimic/module.ex

```diff
@@ -57,7 +57,7 @@ defmodule Mimic.Module do
 
       {:ok, module_name, binary, _warnings} ->
         load_binary(module_name, binary)
-        Binary
+        binary
```

Comment:
This seemed like a typo. Happy to revert if you'd like.

Reply:
nice one! Thanks